UI/js/default.js

```diff
@@ -1,4 +1,6 @@
+let scroolBotton = document.getElementById("arrowP");
 const navbarSlider = document.getElementById('header-sm-sc');
+let rejectBtn = document.getElementsByClassName("reject-btn");
```

Comment:
It doesn't seem like you are changing this or `scroolBotton`. They should be `const`s. Also, I think you meant to write `scrollButton`. Please change those.